src/routes/+layout.server.ts

```diff
@@ -1,7 +1,12 @@
 import type {LayoutServerLoad} from './$types'
 import {getProjects} from '$lib/server/database'
 
-export const load: LayoutServerLoad = () => {
-    const projects = getProjects()
+export const load: LayoutServerLoad = ({cookies}) => {
+    const cachedUserID = cookies.get('userID')
+    const userID = cachedUserID ?? crypto.randomUUID()
+    if (userID !== cachedUserID) {
+        cookies.set('userID', userID)
+    }
```

Comment:
Проще читалось бы так, думаю

```suggestion
    let userID = cookies.get('userID')
    if (!userID) {
        userID = crypto.randomUUID()
        cookies.set('userID', userID)
    }
```